docs/developer_docs.md

```diff
@@ -87,6 +87,11 @@ This plugin integrates the ServiceNow Virtual Agent in Mattermost. It is created
     - [Set up trusted media domains](https://docs.servicenow.com/bundle/quebec-now-intelligence/page/administer/virtual-agent/task/ccif-secure-file-upload.html)
       - [Configuring the form layout](https://docs.servicenow.com/en-US/bundle/sandiego-platform-administration/page/administer/form-administration/concept/configure-form-layout.html)
 
+- **Invalid response**
+    We get an "undefined" response from Virtual Agent API in a specific case. You can read [this](https://www.servicenow.com/community/virtual-agent-nlu-forum/getting-improper-response-from-virtual-agent-bot-integration-api/m-p/255032) for more information.
```

Comment:
May be this is better?
```suggestion
 If we get an "undefined" response from Virtual Agent API in some specific cases. Please refer to [this](https://www.servicenow.com/community/virtual-agent-nlu-forum/getting-improper-response-from-virtual-agent-bot-integration-api/m-p/255032) for more information.
```

Reply:
Changed the second line. But I think the first line if correct as we are telling a developer about the issue.